bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -163,21 +173,39 @@ IActionResult invalidJson(string errorMessageKey, Exception ex = null)
         [HttpGet]
         public async Task<IActionResult> Login(string returnUrl)
         {
-            var context = await _interaction.GetAuthorizationContextAsync(returnUrl);
-            if (context.Parameters.AllKeys.Contains("domain_hint") &&
-                !string.IsNullOrWhiteSpace(context.Parameters["domain_hint"]))
+            try
             {
-                return RedirectToAction(nameof(ExternalChallenge), new
+                var redirectToken = Request.Cookies[SsoRedirectToken.SSO_REDIRECT_COOKIE_NAME];
```

Comment:
I think the flow here looks good/correct for what we're going for, so that's good, however we don't want to use cookies, you'll likely end up wanting to pull this token out of `context.Parameters` given the other feedback and implementation details.